utils/kfp_client.py

```diff
@@ -0,0 +1,55 @@
+# type: ignore
+import warnings
+
+from kfp import Client
+from kubernetes.client import CustomObjectsApi
+from kubernetes.client.configuration import Configuration
+from kubernetes.client.exceptions import ApiException
+from kubernetes.config import list_kube_config_contexts
+from kubernetes.config.kube_config import load_kube_config
+
+
+def get_kfp_client():
+    config = Configuration()
+    load_kube_config(client_configuration=config)
```

Comment:
I believe `load_kube_config()` could fail here, should we wrap this in a try/expect block? Should we fall back to in_cluster config?

Reply:
Ok for not to falling back to in_cluster. How about the try/expect on the `load_kube_config()` now?

Reply:
Done. 👍